runners/apex/src/main/java/org/apache/beam/runners/apex/translation/operators/ApexGroupByKeyOperator.java

```diff
@@ -197,7 +198,7 @@ public void outputWindowedValue(
           }
         },
         NullSideInputReader.empty(),
-        null,
```

Comment:
Note to self -- it's simply a bug fixup on prior CL and does not make sense in isolation.